### PR TITLE
Add TC020 - veryfing infobox's image on category page

### PR DESCRIPTION
**Pull request by nikodamn**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/category/CategoryPageObject.java [hunk 1]
```diff
@@ -30,4 +36,21 @@ public void verifyCategoryPageTitle(String categoryLinkName) {
         );
         Assertion.assertEquals("Category:" + title, categoryLinkName);
     }
+
+    public int getArticleIndexInGalleryByName(String articleName) {
+        int index = -1;
+        for(int i=0; i<categoryGalleryItemsName.size(); i++)
+        {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/category/CategoryPageObject.java]: MAJOR Move this left curly brace to the end of previous line of code. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ALeftCurlyBraceEndLineCheck)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/category/CategoryPageObject.java [hunk 2]
```diff
@@ -30,4 +36,21 @@ public void verifyCategoryPageTitle(String categoryLinkName) {
...
+            if(articleName.equals(categoryGalleryItemsName.get(i).getText()))
+            {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/category/CategoryPageObject.java]: MAJOR Move this left curly brace to the end of previous line of code. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ALeftCurlyBraceEndLineCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 2 major

Watch the comments in this conversation to review them.

**Comment by nikodamn** [comment]
@RodriGomez take a look, please

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/PortableInfoboxPageObject.java [hunk 3]
```diff
@@ -63,14 +59,22 @@
...
-  @FindBy(css = ".pi-title")
-  private WebElement titleWrapper;
-  @FindBy(css = "body")
-  private WebElement bodyElement;
-  @FindBy(css = ".header-title")
-  private WebElement headerTitle;
+  @FindBy(css = "#articleCategories .category a")
+  private List<WebElement> categories;
+  @FindBy(css = ".pi-navigation a[href*='redlink']")
+  private List<WebElement> pInfoRedlLink;
```
Comment (RodriGomez) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/PortableInfoboxPageObject.java]: To make it consistent with the following selector "pInfoInternalLinks", lets call it "pInfoRedLinksList" and also add List to "pInfoInternalLinks"

Comment (nikodamn) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/PortableInfoboxPageObject.java]: Should be better now

**Comment by RodriGomez** [comment]
LGTM

**Pull request merged**